xlogger/hooks/bugsnag.go

```diff
@@ -0,0 +1,52 @@
+package hooks
```

Comment:
Where's the config of bugsnag coming from?

Reply:
Bugsnag is configured globally so it can be done from anywhere, but it is a good idea to make this part of this NewBugsnagHook function: 

	bugsnag.Configure(bugsnag.Configuration{
		APIKey:       cfg.APIKey,
		ReleaseStage: cfg.ReleaseStage,
		ProjectPackages: []string{
			"main",
			"akzo/marketplace-bff/pkg/api",
			"akzo/marketplace-bff/pkg/config",
			"akzo/marketplace-bff/pkg/service/crm",
			"akzo/marketplace-bff/pkg/storage/db",
			"akzo/marketplace-bff/pkg/storage/db/models",
			"akzo/marketplace-bff/pkg/shared/cache",
		},
		NotifyReleaseStages: []string{"test", "qa", "preprod", "prod"},
	})

Reply:
I added an optional bugsnag configuration to the hook